nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/DeConv2D.java

```diff
@@ -73,6 +73,19 @@ public DeConv2D(@NonNull INDArray input, @NonNull INDArray weights, INDArray bia
         this(wrapFilterNull(input, weights, bias), wrapOrNull(output), config);
     }
 
+    public DeConv2D(INDArray layerInput, INDArray weights, DeConv2DConfig deConv2DConfig) {
+        this(wrapFilterNull(layerInput, weights), wrapOrNull(null), deConv2DConfig);
+    }
+
+    public DeConv2D(INDArray layerInput, INDArray weights, INDArray bias, DeConv2DConfig deConv2DConfig) {
+        this(wrapFilterNull(layerInput, weights, bias), wrapOrNull(null), deConv2DConfig);
+    }
+
+    public DeConv2D(INDArray inputs, DeConv2DConfig deConv2DConfig) {
+        this(wrapFilterNull(inputs), wrapOrNull(null), deConv2DConfig);
+    }
```

Comment:
Don't do this: `wrapOrNull(null)`
Just pass null instead.
That's equivalent to `if(null == null){ return null } else { return new INDArray[]{null}`

Also weights are not optional, so this constructor needs to be deleted: `public DeConv2D(INDArray inputs, DeConv2DConfig deConv2DConfig) `

So this whole block should look like:
```
    public DeConv2D(INDArray layerInput, INDArray weights, DeConv2DConfig deConv2DConfig) {
        this(layerInput, weights, null, deConv2DConfig);
    }

    public DeConv2D(INDArray layerInput, INDArray weights, INDArray bias, DeConv2DConfig deConv2DConfig) {
        this(wrapFilterNull(layerInput, weights, bias), null, deConv2DConfig);
    }
```